### ISSUE TITLE
'struct objc_object' is not callable (macOS Sierra 10.12.3)

### ISSUE THREAD

**Issue by fdagostino** [luapower/objc#3]
## Enviroment
- macOS Sierra 10.12.3 
- MacBook (Retina, 12-inch, Early 2015)
- Memory 8 GB 1600 MHz DDR3
- LuaJIT 2.0.4
- ZeroBrane Studio 1.50
- luapower/objc (master)

## Description
When running a simple script I get the following output in ZeroBrane Studio or in Terminal Console:
```
Program starting as '"/Applications/ZeroBraneStudio.app/Contents/ZeroBraneStudio/bin/lua.app/Contents/MacOS/lua" -e "io.stdout:setvbuf('no')" "/Users/Francisco/dev/test/test.lua"'.
Program 'lua' started in '/Users/Francisco/dev/test' (pid: 22951).
/Applications/ZeroBraneStudio.app/Contents/ZeroBraneStudio/bin/lua.app/Contents/MacOS/lua: /Users/Francisco/dev/test/test.lua:5: 'struct objc_object' is not callable
stack traceback:
	[C]: in function 'operatingSystemVersionString'
	/Users/Francisco/dev/test/test.lua:5: in main chunk
	[C]: at 0x00001c80
Program completed in 0.11 seconds (pid: 22951).
```
## Repro
```
local objc = require('objc.objc')
objc.load('Foundation')
print(objc.tolua(objc.NSProcessInfo:processInfo():operatingSystemVersionString()))
```

**Comment by capr** [comment]
Sorry for late reply hehe :) I totally missed your issue for like, 8 months... It seems like they "modernized" Cocoa in 10.10 changing a ton of getters to properties. You can read more about it here: 
https://developer.apple.com/library/content/releasenotes/MacOSX/WhatsNewInOSX/Articles/MacOSX10_10.html (section "Frameworks and Framework Technologies")

So on 10.10+ you should do:

`print(objc.tolua(objc.NSProcessInfo:processInfo().operatingSystemVersionString))`

Did you find other issues with objc or gave up on this example?

**Comment by capr** [comment]
(accidentally hit close)

**Comment by capr** [comment]
Fixed (see #4)
